packages/astro/src/integrations/hooks.ts

```diff
@@ -351,6 +355,7 @@ export async function runHookConfigDone({
 						return new URL(normalizedFilename, settings.dotAstroDir);
 					},
 					logger: getLogger(integration, logger),
+					buildOutput: () => settings.buildOutput!, // settings.buildOutput is always set at this point
```

Comment:
Would it be worth making this a getter instead so that users don't have to call a function to get its value?